### DISCUSSION TITLE
Desktop App (Using Tauri)

### DISCUSSION THREAD

**Discussion by trymeouteh** [hoppscotch/hoppscotch#3987] [Ideas]
Please consider making a desktop app for Hoppscotch to allow user to install this development tool as an application on their device

I would recommend using Tauri over Electron for better performance.

I would like to see the desktop app be released in the following ways...

- Windows
   - EXE
   - Scoop
   - Winget
- MacOS
   - DMG
   - Homebrew
- Linux
   - AppImage
   - NIX
   - Flatpak

**Comment by thetronjohnson** [comment]
Hi @trymeouteh we do have a desktop app using Tauri
https://hoppscotch.com/download

ps: the app is still in alpha, we're aiming to roll out a stable release soon
